client/lib/resource/src/router.js

```diff
@@ -56,6 +57,7 @@ class ResourceRouter extends Router {
      */
     createScope(resourceId) {
         Flux.getActions('resource').fetchResource(resourceId);
+        Flux.getActions('resource').fetchScopes(resourceId);
```

Comment:
You probably want to put those to actions together in a `Promise.all` and wait for it, because otherwise

1. you start typing in the form
2. one of these two actions finishes
3. the store updates and emits a change event (it *always* emits a change event, even though it received the same data it already has—a consequence of immutable data structures)
4. the view re-renders completely (because it swaps the whole "body" with `$el.html(Template())`)
5. you lose what you just typed and are in an undesirable emotional state of confusion and madness